unitTests/utils/DomTest.ts

```diff
@@ -31,6 +31,14 @@ export function DomTests() {
         Simulate.removeJQuery();
       });
 
+      it('should properly detect supported event', () => {
```

Comment:
I think it would be better to have separate tests for the happy path (event supported) and the path when unsupported. When one of the test fails, we have a better test description to tell use what happened.